airflow/utils/logging.py

```diff
@@ -57,6 +58,19 @@ def __init__(self):
                 'Please make sure that airflow[s3] is installed and '
                 'the S3 connection exists.'.format(remote_conn_id))
 
+    def exists(self, remote_log_location):
```

Comment:
Would be good to name this something a bit more descriptive:
def log_exists(self, log_location)